test/extended/testdata/test-imagesource-build.yaml

```diff
@@ -89,7 +89,7 @@ items:
           deploymentconfig: imagesourceapp
       spec:
         containers:
-        - image: imagesourceapp
+        - image: " "
```

Comment:
making this an empty string would be another way to validate the fake out of the k8s validation i put in a while back ... maybe make this one empty string, and the one below a single space, to cover both paths